aws_demo_project.yml

```diff
@@ -1,33 +1,36 @@
-schema_version: 0.2
-stock_type: residential
+schema_version: '0.3'
 buildstock_directory: ../resstock  # Relative to this file or absolute
 project_directory: project_national  # Relative to buildstock_directory
 output_directory: ../demo_test_outputs
 weather_files_url: https://data.nrel.gov/system/files/128/ResStock_TMY3.zip
 
 baseline:
-  n_datapoints: 4  # Comment this line out if using a custom buildstock csv file
   n_buildings_represented: 133172057  # Total number of residential dwelling units in contiguous United States, including unoccupied units, resulting from acensus tract level query of ACS 5-yr 2016 (i.e. 2012-2016), using this script: https://github.com/NREL/resstock-estimation/blob/master/sources/spatial/tsv_maker.py.
-  sampling_algorithm: quota # The default resstock sampling algorithm - use precomputed if using the precomputed_sample option
+
+sampler:
+  type: residential_quota # change to residential_quota_downselect to do downselect
+  args:
+    n_datapoints: 4
+    # resample: false # Uncomment and set specify logic you you want to downselect to a subset of the building stock
+    # logic:
+    #   - Geometry Building Type RECS|Single-Family Detached
+    #   - Vacancy Status|Occupied
```

Comment:
Can you move the `resample` arg to after `logic`? It just seems like flow more logically that way, at least to me.
```
    # logic:
    #   - Geometry Building Type RECS|Single-Family Detached
    #   - Vacancy Status|Occupied
    # resample: false # Uncomment and set specify logic you you want to downselect to a subset of the building stock
```
And is it worth prefacing both: `downselect_logic` and `downselect_resample`, so it is clearer that they are linked?

Reply:
The keys can be ordered however you want. I guess I figured that since I was using the `residential_quota_downselect` sampler, it was clear. It's a minor hassle to change, so if it's not a big deal to you, I'd rather not.